utility/gw2wannier90.py

```diff
@@ -0,0 +1,374 @@
+#!/usr/bin/env python2
+#
+# gw2wannier90 interface
```

Comment:
Can you add this text in the comments, for the license (if it's ok to license under GPL)?

```
# This file is distributed as part of the Wannier90 code and
# under the terms of the GNU General Public License. See the
# file `LICENSE' in the root directory of the Wannier90
# distribution, or http://www.gnu.org/copyleft/gpl.txt
#
# The webpage of the Wannier90 code is www.wannier.org
#  
# The Wannier90 code is hosted on GitHub:
#
# https://github.com/wannier-developers/wannier90
```